pre_commit/commands/hook_impl.py

```diff
@@ -175,6 +177,7 @@ def _pre_push_ns(
     'post-merge': 1,
     'post-rewrite': 1,
     'pre-push': 2,
+    'pre-rebase': 1,
```

Comment:
That it should be `2`? Following the documentation it can receive two params: https://git-scm.com/docs/githooks#_pre_rebase